src/xo-app/backup/file-restore/index.js

```diff
@@ -118,9 +118,18 @@ export default class FileRestore extends Component {
       ? <Container>
         <h2>{_('restoreFiles')}</h2>
         {isEmpty(backupInfoByVm)
-          ? _('noBackup')
+          ? <div>
+            <em><Icon icon='info' /> {_('restoreDeltaBackupsInfo')}</em>
+            <div>
+              <a>{_('noBackup')}</a>
+            </div>
+          </div>
           : <div>
-            <em><Icon icon='info' /> {_('restoreBackupsInfo')}</em>
+            <ul>
+              <li><em><Icon icon='info' /> {_('restoreBackupsInfo')}</em></li>
+              <li><em><Icon icon='info' /> {_('restoreDeltaBackupsInfo')}</em></li>
+            </ul>
+
```

Comment:
Bullets of the list seem a bit redundant with the "i" circle icon. That's a minor thing though, just the feeling it's not really nice.